build.gradle

```diff
@@ -163,7 +163,7 @@ dependencyManagement {
         dependency group: 'com.github.jknack', name: 'handlebars', version: '4.2.0'
 
         //CVE-2021-22060
-        dependencySet(group: 'org.springframework', version: '5.2.19.RELEASE') {
+        dependencySet(group: 'org.springframework', version: '5.2.20.RELEASE') {
```

Comment:
this is a quite old CVE, ideally we should be looking into upgrading Spring to the latest version, if not on this at least raising a tech debt ticket.

Reply:
My mistake, cve fix was meant to be this: CVE-2022-22965